README.md

```diff
@@ -1 +1,39 @@
-# esb-connector-amazonlambda
\ No newline at end of file
+# AmazonLambda EI Connector
+
+
+The AmazonLambda Connector allows you to access the REST API of [Amazon Web Service Lambda (AWS Lambda)](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html) ,
+which lets you run code without provisioning or managing servers. With Lambda, you can run code for virtually any type of application or backend service - all with zero administration. Just upload your code  in one of the languages that AWS Lambda supports (currently Node.js, Java, C#, Go and Python) and Lambda takes care of everything required to run and scale your code with high availability. You can set up your code to automatically trigger from other AWS services or call it directly from any web or mobile app. You pay only for the compute time you consume-there is no charge when your code is not running.
+##Compatibility
+
+| Connector Version | Supported WSO2 EI version |
+| ------------- |-------------|
+| 1.0.0    | EI 6.4.0 |
```

Comment:
Add API version to this version table

Reply:
Seems API version is not unique for all method so didn't add here. Note is attached there though for info.